iron-dropdown-scroll-manager.html

```diff
@@ -40,10 +41,27 @@
        */
       elementIsScrollLocked: function(element) {
         var currentLockingElement = this.currentLockingElement;
+        var scrollLocked;
+
+        if (this._hasCachedLockedElement(element)) {
+          return true;
+        }
 
-        return !!currentLockingElement &&
+        if (this._hasCachedUnlockedElement(element)) {
+          return false;
+        }
+
+        scrollLocked = !!currentLockingElement &&
           currentLockingElement !== element &&
-          !currentLockingElement.contains(element);
+          !this._composedTreeContains(currentLockingElement, element);
```

Comment:
Since this method is called from an event handler, might it be possible to use `event.path` to walk the composed tree?